docs/template_plugin/tests/functional/op_reference/logical_or.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+#include <tuple>
+
+#include "base_reference_test.hpp"
+
+using namespace reference_tests;
+using namespace ngraph;
+using namespace InferenceEngine;
+
+struct LogicalOrParams {
+    template <class IT, class OT>
+    LogicalOrParams(const ngraph::PartialShape& input_shape1, const ngraph::PartialShape& input_shape2, const std::vector<IT>& iValues1,
+                    const std::vector<IT>& iValues2, const std::vector<OT>& oValues)
+        : pshape1(input_shape1),
+          pshape2(input_shape2),
+          inType(ngraph::element::boolean),
+          outType(ngraph::element::boolean),
+          inputData1(CreateBlob(ngraph::element::boolean, iValues1)),
+          inputData2(CreateBlob(ngraph::element::boolean, iValues2)),
+          refData(CreateBlob(ngraph::element::boolean, oValues)) {}
+    ngraph::PartialShape pshape1;
+    ngraph::PartialShape pshape2;
+    ngraph::element::Type inType;
+    ngraph::element::Type outType;
```

Comment:
As type of input and output is same, boolean type, you can omit `oType` member. It is only  used in `getTestCaseName` function and it is redundant.

Reply:
Done